### ISSUE TITLE
Allow Users to choose which Data Mines to include in their workspace

### ISSUE THREAD

**Issue by Luis-Fernando-Molina** [Superalgos/Superalgos#212]
This includes how to manage the Data Mines that users are starting to create and would like to open source to be somehow part of the system. Today you are forced to add to your workspace Masters and Sparta, because almost all depends on that, but I am already seeing some users have create new Data Mines which are essentially a package of indicators. To allow them to make their work available to the rest, there is the need to be something that allows users to pick which Data Mines they want to add to their workspace and which ones they don't.

**Comment by Luis-Fernando-Molina** [comment]
This can be done at the same sprint:

https://github.com/Superalgos/Superalgos/issues/213

**Comment by Luis-Fernando-Molina** [comment]
From the Telegram Community 

9808us, [26.04.20 09:02]
Forgive me Julian but I still have details to ask ... 😅😅

9808us, [26.04.20 09:02]
And as I go deeper into your project, I try to understand better how it works from foundations.
There is one aspect that in my opinion is a bit tricky and concerns the way in which Superalgos manages the information saved when the program is launched.
I noticed this problem when I try to reset Superalgos to bring it back to its original state (but maybe it's not a problem, maybe it's wanted it to do so). I would have expected that deleting the main directory (except Data-Storage for obvious reasons I will not delete the folder) and then copying the project again to find myself in an initial state environment as if I had done the download for the first time. Instead it is not so. At first I did not understand why but then I tried more deeper and found the culprit. Here are the steps:
- I install Superalgos for the first time
- The browers open and from the console I see the work begin
- I make changes, like to install a new exchange
- I close everything
So far so good, when I restart the program I find myself exactly where I left off.
Now I try to reset Superagos, delete the folder, copy the original files again, the browser opens, F12 -> Empty Cache and Hard Reset but unexpectedly I find myself where I was and not from the beginning!
I think the "problem" is how the parameters / sessions (that I would have expected would be saved in a folder in Superalgos) are instead saved in the browser database. This involves for example a strange situation:
- Superalgos launch
- The default browser opens
- I make changes
- In parallel I open another browser and go to http: // localhost: 34247
- If it's the first time I find everything reset
I wonder then why not save the database in Superalgos rather than in the browser, because in this way the program is browser-dependent and it does not seem to me the optimal solution

**Comment by Luis-Fernando-Molina** [comment]
9808us, [26.04.20 09:30]

Perfect, in fact naively I looked at the browser as a simple interface and I thought that everything instead "worked" inside Superalgos and its folders. For this reason I did not understand why even if I was erasing everything I still found back to the same point where I had left. This also leads to problems because if for example I use a browser to do a job, and then maybe later I use another browser to do another job in superalgos, then errors may kick in such as lack of dependencies can occur.
Thanks for the wonderful work you are doing and I still hope to do something pleasant by reporting bugs or maybe giving suggestions even if as I said I am not a dev but only a trader looking for a program that makes me independent

9808us, [26.04.20 09:55]

You know ... we traders must have at least 6/8 monitors to works and satisfy ourself 😂😂😂
I'm not sure they will be synchronized (but I didn't try yet)
